src/main/java/com/cloudogu/smeagol/repository/domain/Description.java

```diff
@@ -0,0 +1,55 @@
+package com.cloudogu.smeagol.repository.domain;
+
+import com.google.common.base.Strings;
+
+import java.util.Objects;
+
+/**
+ * Description is simple string value object, which can be empty.
+ */
+public class Description {
+
+    private final String value;
+
+    private Description(String value) {
+        this.value = value;
+    }
+
+    /**
+     * Returns the string representation of the name.
+     *
+     * @return string representation
+     */
+    public String getValue() {
+        return value;
+    }
+
+    /**
+     * Creates a description from a string.
+     *
+     * @param value string representation of a description
+     *
+     * @return repository name
+     */
+    public static Description valueOf(String value) {
+        return new Description(Strings.nullToEmpty(value));
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AS00122)